### ISSUE TITLE
Tags and delete_on_termination for nova volume attachments

### ISSUE THREAD

**Issue by nikParasyr** [gophercloud/gophercloud#2176]
Nova allows to set tags and delete_on_termination when attaching a volume to an instance.
The related docs: https://docs.openstack.org/api-ref/compute/?expanded=attach-a-volume-to-an-instance-detail#attach-a-volume-to-an-instance

**Comment by kayrus** [comment]
@nikParasyr this feature is already supported in `bootfromvolume` extension:

```sh
$ grep -r DeleteOn openstack/compute/v2/extensions/bootfromvolume/requests.go
        // DeleteOnTermination specifies whether or not to delete the attached volume
        DeleteOnTermination bool `json:"delete_on_termination"`
```

**Comment by nikParasyr** [comment]
@kayrus thanks for the input. If im not mistaken this is a bit different. It is regarding the `os-volume_attachments` action based on docs, which is this one: https://github.com/gophercloud/gophercloud/tree/master/openstack/compute/v2/extensions/volumeattach. Both tag and delete_on_termination are missing. Might be missing something though

**Comment by kayrus** [comment]
@nikParasyr `os-volume_attachments` actually is used only, when you make an action against the existing VM. When you create a new VM, the `server` endpoint is used inside the `servers` package. As an example take a look on terraform source code:
https://github.com/terraform-provider-openstack/terraform-provider-openstack/blob/23b3d55fe9d50dbede92f00d0a0f88fa011f4170/openstack/resource_openstack_compute_instance_v2.go#L1215

**Comment by nikParasyr** [comment]
> os-volume_attachments actually is used only, when you make an action against the existing VM. 

Yes. This issue is to cover the downstream https://github.com/terraform-provider-openstack/terraform-provider-openstack/issues/1262 issue for the `openstack_compute_volume_attach_v2` resource which uses the [volumeattach](https://github.com/terraform-provider-openstack/terraform-provider-openstack/blob/23b3d55fe9d50dbede92f00d0a0f88fa011f4170/openstack/resource_openstack_compute_volume_attach_v2.go#L9) package. 

> UPD: sorry for a confusion. probably you mean the update action, which appeared in Kilo release?

No, i'm referring to:
tag => added on 2.49 =>https://docs.openstack.org/nova/latest/reference/api-microversion-history.html#id45
delete_on_termination => added on 2.79 =>  https://docs.openstack.org/nova/latest/reference/api-microversion-history.html#maximum-in-train

To me, after looking a bit, this functionality seems to:
- not correlate with the bootfromvolume
- not supported yet

**Comment by nikParasyr** [comment]
Closing. Implemented in #2177
